dotnet/src/Microsoft.Agents.AI.Hosting.OpenAI/Responses/AIAgentResponsesProcessor.cs

```diff
@@ -139,6 +149,7 @@ private async IAsyncEnumerable<SseItem<StreamingResponseEventBase>> GetStreaming
                     CreatedAt = update.CreatedAt,
                     RawRepresentation = update.RawRepresentation
                 };
+                var messageId = chatMessage.MessageId ?? "<null>";
```

Comment:
Why is this using `"<null>"`? Is that just an arbitrary sentinel to use to group together all messages without IDs?